bundles/org.eclipse.ui.workbench/eclipseui/org/eclipse/ui/internal/WindowsDefenderConfigurator.java

```diff
@@ -340,16 +340,12 @@ public static String createAddExclusionsPowershellCommand(String extraSeparator)
 		// https://learn.microsoft.com/en-us/powershell/module/defender/add-mppreference?view=windowsserver2019-ps
 		// https://learn.microsoft.com/en-us/powershell/module/defender/get-mppreference?view=windowsserver2019-ps
 		//
-		// For .NET's stream API called LINQ see:
-		// https://learn.microsoft.com/en-us/dotnet/api/system.linq.enumerable
-		String excludedPaths = paths.stream().map(Path::toString).map(p -> '"' + p + '"')
+		// For detailed explanations about how to add new exclusions see:
+		// https://learn.microsoft.com/en-us/powershell/module/defender/add-mppreference?view=windowsserver2019-ps
+
+		String excludedPaths = paths.stream().map(Path::toString).map(p -> "\"" + p + "\"")
```

Comment:
Please just keep the two double-quote char-literals instead of converting them to string-literals as the latter would require NON-NLS comments (i.e. `//$NON-NLS-1$//$NON-NLS-2$`) and this just adds unnecessary complexity:
```suggestion
		String excludedPaths = paths.stream().map(Path::toString).map(p -> '"' + p + '"')
```

Reply:
Applied this now.